src/components/user-dropdown-disclosure/index.tsx

```diff
@@ -91,6 +98,9 @@ const UserDropdownDisclosure = ({
 				</>
 			) : null}
 			{items.map(renderItem)}
+			{shouldRenderThemeSwitcher ? (
+				<UserDropdownDisclosureThemeSwitcher />
+			) : null}
```

Comment:
🙋🏻 This is just a question about the shift to using an assembly branch -- Is dark mode getting released when that assembly branch is merged? Or when the the config flag is flipped? If it's going to be when the assembly branch is merged, we could probably avoid the conditionals now! 💭

Reply:
Good point, Well also flip the config in the assembly and merge the assembly for the release. I can remove the conditional here though.

Reply:
Oh yes! somehow I missed this comment this morning. Will update now

Reply:
So actually I still need the conditional for the themed route, and I'll leave the config flag just for clarity at the moment.